performance-test/scripts/Esignet_Mockida_Test_Script.jmx

```diff
@@ -0,0 +1,1511 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.3">
+  <hashTree>
+    <TestPlan guiclass="TestPlanGui" testclass="TestPlan" testname="Test Plan" enabled="true">
+      <stringProp name="TestPlan.comments"></stringProp>
+      <boolProp name="TestPlan.functional_mode">false</boolProp>
+      <boolProp name="TestPlan.tearDown_on_shutdown">true</boolProp>
+      <boolProp name="TestPlan.serialize_threadgroups">false</boolProp>
+      <elementProp name="TestPlan.user_defined_variables" elementType="Arguments" guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
+        <collectionProp name="Arguments.arguments"/>
+      </elementProp>
```

Comment:
Create Identities in Mock Identity System (Setup):
1. Create Mock identity Endpoint - do we need the individualBiometrics in the request ? since we provide the photo separately ?
2. Store Identifier and Password and Full Names To File - can you store the phone number and email id also in the csv file ?
3. Variable fullNames to be renamed to fullName

Reply:
@vishwa-vyom 
1. We are unable to generate mock identities while removing individualBiometrics field.
2. I have updated Store Identifier and Password and Full Names To File by adding email id, phone number
3. Changed to fullNames to fullName

Reply:
all 3 points are ok